src/ui/public/vislib/visualizations/gauges/simple.js

```diff
@@ -217,8 +217,9 @@ export function SimpleGaugeProvider() {
             const percentage = Math.round(100 * (d.y - min) / (max - min));
             return `${percentage}%`;
           }
-          if (d.aggConfig) {
-            return d.aggConfig.fieldFormatter('text')(d.y);
+          if (d.aggConfigResult && d.aggConfigResult.aggConfig) {
```

Comment:
Would this be simpler to just use lodash `has`? Also, the `if` check in gauges/meter.js is looking for something different than this check but they both use d.aggConfigResult.aggConfig